src/views/CreateView.vue

```diff
@@ -343,6 +347,17 @@ export default {
         this.isFormSubmitted = true
       }
     }
+  },
+  mounted() {
+    const fetchExercises = async () => {
+      const { data } = await supabase
+        .from('Exercises')
+        .select()
+
+      this.allExercises = data
+    }
+
+    fetchExercises()
```

Comment:
this should be awaited so `mounted` should be async

Reply:
I believe I corrected that, not sure though. Should I now presumed that not only the functions is supposed to be asynchronous but the action of calling it as well?